src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/on_driver_distraction_notification.cc

```diff
@@ -144,6 +158,32 @@ void OnDriverDistractionNotification::Run() {
   std::for_each(applications.begin(), applications.end(), processor);
 }
 
+bool OnDriverDistractionNotification::AddLockScreenDismissalWarningToMessage(
+    ns_smart_device_link::ns_smart_objects::SmartObjectSPtr notification_so,
```

Comment:
@IGapchuk can you pass OnDriverDistractionNotification by reference?

Reply:
@LuxoftAKutsan Fixed in 048fe1415fce72016340c817c56367238ee052f6